docs/serialize.md

```diff
@@ -5,38 +5,71 @@ Defines custom serialization behavior for a property.
 
 @signature `Boolean`
 
-Specifies if the property should be serialized.  By default, all properties except for
-ones with defined [can-define.types.get getters] are serialized. Prevent a property
-from being serialized like:
+  Specifies if the property should be serialized.  By default, all properties except for
+  ones with defined [can-define.types.get getters] are serialized. Prevent a property
+  from being serialized like:
 
-```js
-{
-	propertyName: {
-		serialize: false
-	}
-}
-```
+  ```js
+  import {DefineMap} from "can";
 
-Make a [can-define.types.get getter] property part of the serialized result like:
+  const myMap = DefineMap.extend({
+    propertyName: {
+      serialize: false
+    },
+    secondPropertyName: "string"
+  });
 
-```js
-{
-	propertyName: {
-		get: function() { /* ... */ },
-		serialize: true
-	}
-}
-```
+  const map = new myMap({ propertyName: "foobar", secondPropertyName: "bar" });
+
+  console.log( map.serialize() ); //-> {secondPropertyName: "bar"}
+  ```
+  @codepen
+
+  Make a [can-define.types.get getter] property part of the serialized result like:
+
+  ```js
+  import {DefineMap} from "can";
+
+  const myMap = DefineMap.extend({
+    propertyName: {
+      get() { return "test"; },
+      serialize: true
+    }
+  });
+
+  const map = new myMap();
+
+  console.log( map.serialize() ); //-> { propertyName: "test" }
+  ```
+  @codepen
 
 @signature `serialize( currentValue, propertyName )`
 
-Specifies the serialized value of a property.
+	Specifies the serialized value of a property.
```

Comment:
This indentation is causing:

![image](https://user-images.githubusercontent.com/78602/45120582-2c00f780-b124-11e8-80d8-7287418caedf.png)